packages/admin/src/Http/Controllers/AssetController.php

```diff
@@ -79,6 +79,6 @@ protected function pretendResponseIsFile(string $path, string $contentType)
             'Expires' => $this->getHttpDate($expires),
             'Cache-Control' => $cacheControl,
             'Last-Modified' => $this->getHttpDate($lastModified),
-        ]);
+        ])->send();
```

Comment:
Could you put each chained method after `response()` on its own line?
Mark as ready when done. 😊